sdk/applicationinsights/ci.yml

```diff
@@ -20,4 +20,4 @@ pr:
 jobs:
 - template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
   parameters:
-    BuildTargettingString: azure-mgmt-applicationinsights,azure-applicationinsights
\ No newline at end of file
+    ServiceDirectory: sdk/applicationinsights/
```

Comment:
Any good reason we cannot omit the `sdk/` and only provide the service folder name? Just like we did in .NET.

Reply:
Can't think of a good reason. I'll make that change.